homeassistant/components/gryfsmart/__init__.py

```diff
@@ -0,0 +1,51 @@
+"""The Gryf Smart integration."""
+
+from __future__ import annotations
+
+from pygryfsmart.api import GryfApi
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import CONF_API, CONF_COMMUNICATION, CONF_DEVICE_DATA, CONF_PORT, DOMAIN
+
+_PLATFORMS: list[Platform] = [
+    Platform.LIGHT,
+]
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
```

Comment:
extend the config entry type and add the data you put in entry.runtime_data there